### ISSUE TITLE
Correct use of Alt Tags/Image Descriptions 

### ISSUE THREAD

**Issue by lukeware** [suwebdev/purple-haze#2]
**User Story:** [Accessibility #1](https://github.com/suwebdev/purple-haze/wiki/User-Stories:-Accessibility)

**Size:** Large

Reason for adding to Sprint: As we add content to our page, we must ensure that alt tags are used appropriately. Images should be evaluated for both content and function in terms of accessibility. To do this I will be following the [WEBAIM guidance](http://webaim.org/techniques/alttext/) to ensure that alt tags are utilized or left null depending on the value of the image. 

**Priority:** High

**Comment by lukeware** [comment]
Task-branch name: bio-page

Notes: Added a Maya image on this page. Because I captioned the image, I have left the alt tags as null.
